src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -0,0 +1,135 @@
+//            public static extern uint                  Dali_Toolkit_DevelControl_AccessibilityRelations_Size(Tizen.NUI.BaseComponents.View.AddressCollection arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_AccessibilityRelations_RelationSize")]
+            public static extern uint                  Dali_Toolkit_DevelControl_AccessibilityRelations_RelationSize(Tizen.NUI.BaseComponents.View.AddressCollection arg1, int relation);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_AccessibilityRelations_At")]
+            public static extern string                Dali_Toolkit_DevelControl_AccessibilityRelations_At(Tizen.NUI.BaseComponents.View.AddressCollection arg1, int rel, int pos, int id);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_delete_AccessibilityRelations")]
+            public static extern void                  Dali_Toolkit_DevelControl_delete_AccessibilityRelations(Tizen.NUI.BaseComponents.View.AddressCollection arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_ClearAccessibilityRelations")]
+            public static extern void                  Dali_Toolkit_DevelControl_ClearAccessibilityRelations(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_AppendAccessibilityAttribute")]
+            public static extern void                  Dali_Toolkit_DevelControl_AppendAccessibilityAttribute(global::System.Runtime.InteropServices.HandleRef arg1, string arg2, string arg3);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_RemoveAccessibilityAttribute")]
+            public static extern void                  Dali_Toolkit_DevelControl_RemoveAccessibilityAttribute(global::System.Runtime.InteropServices.HandleRef arg1, string arg2);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_ClearAccessibilityAttributes")]
+            public static extern void                  Dali_Toolkit_DevelControl_ClearAccessibilityAttributes(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_SetAccessibilityReadingInfoType")]
+            public static extern void                  Dali_Toolkit_DevelControl_SetAccessibilityReadingInfoType(global::System.Runtime.InteropServices.HandleRef arg1, Tizen.NUI.BaseComponents.View.ReadingInfoTypes arg2);
```

Comment:
after consideration please change ReadingInfoType to AccessibilityReadingInfoType